listener.py

```
@@ -138,42 +104,43 @@ def main() -> None:
+            member = route_mention(context)
+            print(
+                f"Mention {mention.id} routed to {member.profile.id} ({member.profile.kind})",
+                flush=True,
+            )
             try:
-                client.create_tweet(
-                    text=grok_reply[:280],
-                    in_reply_to_tweet_id=mention.id,
-                )
+                reply = member.handle_mention(context)
             except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)